templates/config/puppet.nft.epp

```diff
@@ -26,6 +26,6 @@ include "custom-*.nft"
 include "inet-filter.nft"
 <% } -%>
 <% if $nat { -%>
-include "ip-nat.nft"
-include "ip6-nat.nft"
+include "ip-<%= $::nftables::nat_table_name %>.nft"
+include "ip6-<%= $::nftables::nat_table_name %>.nft"
```

Comment:
Do we need to reference top scope here? Afaik this still got deprecated unless explicitly needed.

Reply:
yes. I didn't review the whole PR due to the size. tl;dr use `$nat_table_name` if the scope fits, otherwise `$nftables::nat_table_name`.

Reply:
Thanks, i've changed the variable to '$nftables::nat_table_name'

Reply:
LGTM. It could do with a squash.

Reply:
changes squashed as requested